.editorconfig

```diff
@@ -0,0 +1,15 @@
+[*]
+charset=utf-8
+end_of_line=lf
+insert_final_newline=false
+indent_style=tab
+tab_width=4
+
+[{phpunit.xml.dist,*.jhm,*.xslt,*.xul,*.rng,*.xsl,*.xsd,*.ant,*.svg,*.tld,*.fxml,*.jrxml,*.xml,*.jnlp,*.wsdl}]
+indent_style=space
+indent_size=4
```

Comment:
If the `ident_size` is always the same, then why it can't be moved to `[*]` section?

Reply:
This was autogenerated by PHPStorm.

Reply:
I don't trust auto-generated stuff. Better to look deeper into file format itself and create content that would be most useful.